Gemfile

```diff
@@ -31,7 +31,6 @@ gem 'enumerize' # Mongoid doesn't have enum out of the box, so we get it here
 # ...but hopefully soon it will be postgres
 gem 'pg', '~> 1.2'
 gem 'paper_trail', '~> 10.3'
-gem 'paper_trail-globalid'
```

Comment:
We don't actually need this; we rolled our own created_by stuff and set up a relation of `version.user`, so we can remove this from the gemfile.